web/cobrands/tfl/assets.js

```diff
@@ -266,8 +266,20 @@ var rw_stylemap = new OpenLayers.StyleMap({
     })
 });
 
+function format_date(date) {
```

Comment:
Make the one in roadworks.js a function that can be called externally?

Reply:
Good idea, will do.